frontend/src/components/AvailabilitySelector/index.js

```diff
@@ -118,3 +118,7 @@ AvailabilitySelector.propTypes = {
   onChange: propTypes.func.isRequired,
 };
 
+// Get a default value for this component (which depends on the timezon)
+AvailabilitySelector.default = () => {
+  return localToUtc('NNNNNNNNNYYYYYYYYNNNNNNN');
+}
```

Comment:
You could also write it like this, but maybe it'd make more sense to make this the `timeAvailability` default prop:
```
AvailabilitySelector.defaultProps = {
  onChange: localToUtc('NNNNNNNNNYYYYYYYYNNNNNNN');
}
```

Reply:
I've probably solved this the wrong way.  It's a default value for the `timeAvailability` property, rather than `onChange`.  I'm not sure why I made it a callable, rather than just a property.  The reason I didn't use `defaultProps` is that the default needs to be included in the initial `participant` object, so that it's set properly in the REST API request if the user doesn't touch the DOM components.

Maybe a constant would be clearer, since it can give a name to the value:

`import { AvailabilitySelector, LOCAL_NINE_TO_FIVE } from '...';`

I'll give that a shot.